core/gears.lua

```diff
@@ -2526,7 +2526,12 @@ if (DetailsFramework.IsWotLKWow()) then
 		for i = 1, (MAX_TALENT_TABS or 3) do
 			if (i <= numTabs) then
 				--tab information
-				local name, iconTexture, pointsSpent, fileName = GetTalentTabInfo (i)
+				local id, name, description, iconTexture, pointsSpent, fileName
+				if DetailsFramework.IsCataWow() then
+					id, name, description, iconTexture, pointsSpent, fileName = GetTalentTabInfo(i)

```

Comment:
According to BlizzardInterfaceCode the return values for `GetTalentTabInfo` on cata beta are `local id, name, description, icon, pointsSpent, background, previewPointsSpent, isUnlocked` This change just pulls the same values we were originally due to the new argument count.